HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc

```diff
@@ -168,122 +168,127 @@ void HLTBTagPerformanceAnalyzer::dqmBeginRun(const edm::Run& iRun, const edm::Ev
+  //get triggerResults
+  Handle<TriggerResults> TriggerResulsHandler;
+  Handle<reco::JetFlavourMatchingCollection> h_mcPartons;
+  if ( hlTriggerResults_Label.empty() || hlTriggerResults_Label == "NULL" ) 
 	{
-		edm::LogInfo("NoTriggerResults") << "TriggerResults ==> Empty";
-		return;
+      edm::LogInfo("NoTriggerResults") << "TriggerResults ==> Empty";
+      return;
 	}
-	iEvent.getByToken(hlTriggerResults_, TriggerResulsHandler);
-	if (TriggerResulsHandler.isValid())   trigRes=true;
-	if ( !trigRes ) { edm::LogInfo("NoTriggerResults") << "TriggerResults ==> not readable"; return;}
-	const TriggerResults & triggerResults = *(TriggerResulsHandler.product());
+  iEvent.getByToken(hlTriggerResults_, TriggerResulsHandler);
+  if (TriggerResulsHandler.isValid())   trigRes=true;
+  if ( !trigRes ) { edm::LogInfo("NoTriggerResults") << "TriggerResults ==> not readable"; return;}
+  const TriggerResults & triggerResults = *(TriggerResulsHandler.product());
 	
-	//get partons
-	if (m_mcMatching && !m_mcPartons_Label.empty() && m_mcPartons_Label != "NULL" ) {
-		iEvent.getByToken(m_mcPartons, h_mcPartons);
-		if (h_mcPartons.isValid()) MCOK=true;
-	}
+  //get partons
+  if (m_mcMatching && !m_mcPartons_Label.empty() && m_mcPartons_Label != "NULL" ) {
+    iEvent.getByToken(m_mcPartons, h_mcPartons);
+    if (h_mcPartons.isValid()) MCOK=true;
+  }
 
-	//fill the 1D and 2D DQM plot
-	Handle<reco::JetTagCollection> JetTagHandler;
-	for (unsigned int ind=0; ind<hltPathNames_.size();ind++) {
-		bool BtagOK=false;
-		JetTagMap JetTag;
-		if ( !_isfoundHLTs[ind]) continue;	//if the hltPath is not in the event, skip the event
-		if ( !triggerResults.accept(hltPathIndexs_[ind]) ) continue;	//if the hltPath was not accepted skip the event
+  //fill the 1D and 2D DQM plot
+  Handle<reco::JetTagCollection> JetTagHandler;
+  for (unsigned int ind=0; ind<hltPathNames_.size();ind++) {
+    bool BtagOK=false;
+    JetTagMap JetTag;
+    if ( !_isfoundHLTs[ind]) continue;	//if the hltPath is not in the event, skip the event
+    if ( !triggerResults.accept(hltPathIndexs_[ind]) ) continue;	//if the hltPath was not accepted skip the event
 		
-		//get JetTagCollection
-		if (!JetTagCollection_Label[ind].empty() && JetTagCollection_Label[ind] != "NULL" )
-		{
-			iEvent.getByToken(JetTagCollection_[ind], JetTagHandler);
-                        iEvent.getByToken(shallowTagInfosTokenPf_, shallowTagInfosPf);
-                        iEvent.getByToken(shallowTagInfosTokenCalo_, shallowTagInfosCalo);
-			if (JetTagHandler.isValid())   BtagOK=true;
-		}
+    //get JetTagCollection
+    if (!JetTagCollection_Label[ind].empty() && JetTagCollection_Label[ind] != "NULL" )
+      {
+        iEvent.getByToken(JetTagCollection_[ind], JetTagHandler);
+        iEvent.getByToken(shallowTagInfosTokenPf_, shallowTagInfosPf);
+        //                        iEvent.getByToken(shallowTagInfosTokenCalo_, shallowTagInfosCalo);
```

Comment:
@andrzejnovak you introduced shallowTagInfosCalo in #24316 but I do not see where it is used, could you please clarify?

Reply:
@fabiocos It can be left commented out for now. However, we use both Pf and Calo at times for troubleshooting.

Reply:
@andrzejnovak but you use them where sorry? I see no other place in the analyzer where this collection is accessed...

Reply:
@fabiocos  interchangeably, as committed they are not used